tests/apple/resources/enums/test_certificate_type.py

```diff
@@ -10,3 +13,147 @@ def test_from_profile_type(profile_type: ProfileType):
     """
     certificate_type = CertificateType.from_profile_type(profile_type)
     assert isinstance(certificate_type, CertificateType)
+
+
+@pytest.mark.parametrize("certificate_type", list(CertificateType))
+def test_resolve_using_literal_certificate_type(certificate_type: CertificateType):
+    """
+    Check that type can be resolved when CertificateType literal instance is passed via
+    `certificate_types` keyword argument. Expected result is to have a list that contains
+    only the passed type.
+    """
+    resolved_types = CertificateType.resolve(certificate_types=certificate_type)
+    assert resolved_types == [certificate_type]
```

Comment:
Shouldn't it be
```suggestion
    assert resolved_types == certificate_type
```
instead?

Since `certificate_type` is already a list of all possible certificate types?

Reply:
Nope, we parametrize over the values in `CertificateType` enumeration. Each `certificate_type` is an instance of `CertificateType`. Something like
```python
>>> class E(Enum):
...   A = 'A'
...   B = 'B'
...
>>> list(E)
[<E.A: 'A'>, <E.B: 'B'>]
```